### PR TITLE
initial commit for fetch-data

### PR DESCRIPTION
**Pull request by lperryp**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/25247

## Description
A docker image for a new automation for fetching data

### CODE REVIEW

File: docker/fetch-data/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+FROM demisto/python3:3.8.3.9324
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
+  && apk add --no-cache python3-dev libstdc++  \
```
Comment (glicht) [inline docker/fetch-data/Dockerfile:7]: These should be installed as part of the .build-dependencies so they are removed when the step completes. Same goes for g++.

Comment (lperryp) [reply docker/fetch-data/Dockerfile:7]: Thanks, when I have it like this:
```
RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git python3-dev libstdc++ g++ \
  && ln -s /usr/include/locale.h /usr/include/xlocale.h \
  && pip3 install --no-cache-dir -r requirements.txt \
  && apk del .build-dependencies
```

I'm having problems importing Pandas 
` ImportError: Error loading shared library libstdc++.so.6: No such file or directory (needed by /usr/local/lib/python3.8/site-packages/pandas/_libs/window/aggregations.cpython-38-x86_64-linux-gnu.so)`

Comment (glicht) [reply docker/fetch-data/Dockerfile:7]: I see. So you should have only libstdc++ on a separate "apk add"

**Comment by lperryp** [comment]
@glicht  - changed accordingly your previous comment, do I need to commit also .dependabot/config.yml (currently it's in the PR) 
Thanks

**Comment by lperryp** [comment]
@glicht ready from my side

File: docker/fetch-data/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+FROM demisto/python3:3.8.3.9324
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git  g++ \
+  && apk add --no-cache python3-dev libstdc++  \
```
Comment (glicht) [inline docker/fetch-data/Dockerfile:7]: Try moving `python3-dev` to the line above. It should be part of the `.build-dependencies`

Comment (lperryp) [reply docker/fetch-data/Dockerfile]: Thanks missed it, done

**Comment by lperryp** [comment]
> Excellent. Your last commit reduced the image to: 136MB

That's great, thanks for going with me through the process

**Pull request merged**